tests/internal/src/object_library_test/base_object.cpp

```diff
@@ -0,0 +1,19 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC.
+//
+// Produced at the Lawrence Livermore National Laboratory
+//
+// LLNL-CODE-725085
+//
+// All rights reserved.
+//
+// This file is part of BLT.
+//
+// For additional details, please also read BLT/LICENSE.
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+#include "base_object.hpp"
+
+int base_number()
+{
+    return 2;
+}
```

Comment:
Missing a trailing newline 😄

Reply:
Note that this actually triggers a warning in PGI, and thus will not compile with `-Werror`